### PR TITLE
displaying cards 

### PR DESCRIPTION
**Pull request by anniepank**

### CODE REVIEW

File: src/screens/user/user.js [hunk 1]
```diff
@@ -2,17 +2,381 @@ import './user.css';
...
     const screen = createElementsFromString(template());
     target.appendChild(screen);
+
+    let propsForCards = createPropsForCards(userOrders);
+
+    let currentDate = new Date();
+    currentDate = clearHours(currentDate);
+
+    let indexOfCheckedCard = 0;
+    for (let i = 0; i < VISIBLE_NUMBER_OF_CARDS + 1; i++) {
```
Comment (armenbadalyan) [inline src/screens/user/user.js]: Why VISIBLE_NUMBER_OF_CARDS + 1

File: src/screens/user/user.js [hunk 2]
```diff
@@ -2,17 +2,381 @@ import './user.css';
...
+  const today = new Date();
+
+  return {
+    weekday: days[date.getDay()],
+    date: date.toDateString(),
+    active: (date.getDay() >= today.getDay()) && (date.getDate() - today.getDate() < 7),
+  };
+}
+
+function createPropsForCards() {
```
Comment (armenbadalyan) [inline src/screens/user/user.js]: break down this function into smaller functions

File: src/screens/user/user.js (hunk 1 above)
Comment (anniepank) [reply src/screens/user/user.js]: because of Sunday

File: src/screens/user/user.js [hunk 3]
```diff
@@ -339,39 +362,25 @@ export default class UsersScreen {
...
     target.appendChild(screen);
 
-    let propsForCards = createPropsForCards(userOrders);
+    const propsForCards = createPropsForCards(userOrders);
 
-    let currentDate = new Date();
-    currentDate = clearHours(currentDate);
+    propsForCards.forEach((props) => {
+      const card = new Card(target.querySelector('.menus-cards-container'), props);
+      const containerForCard = document.createElement('div');
```
Comment (armenbadalyan) [inline src/screens/user/user.js]: Why is this div needed?

File: src/components/userContent/cardTemplate/card.js [hunk 4]
```diff
@@ -5,32 +5,31 @@ import OrderItem from './orderItem/orderItem';
...
-        this.createOrderItem(target, props.orders[i]);
-      }
-    } else if (props.orders.length > 0 && props.orders.length <= MAX_VISIBLE_ITEMS && props.header.active) {
-      for (const order of props.orders) {
-        this.createOrderItem(target, order);
-      }
-    }
+    target.innerHTML = '';
+    const cardItem = target.appendChild(createElementsFromString(cardTemplate));
+    this.insertCardContent(cardItem, props, target);
```
Comment (armenbadalyan) [inline src/components/userContent/cardTemplate/card.js]: missing return

**Pull request merged**